Marlin/src/pins/stm32/pins_MKS_ROBIN_MINI.h

```diff
@@ -112,7 +112,7 @@
 #define LCD_BACKLIGHT_PIN  PD13
 #define FSMC_CS_PIN        PD7    // NE4
 #define FSMC_RS_PIN        PD11   // A0
-#define TOUCH_CS           PC2
+#define TOUCH_CS_PIN       PC2
```

Comment:
We could add `#if ENABLED(TOUCH_BUTTONS)` to the robin pins, if we think other LCDs might be used with these boards in the future.